### PR TITLE
Tig 1583 featured program slice

### PR DESCRIPTION
**Pull request by klancashire**
Task: https://app.clickup.com/t/9009201449/TIG-1583
Screenshot 2024-07-16 at 3 29 04 PM

Design: https://www.figma.com/design/HUltf86AoUz1Q3VNW5Wz29/Taking-It-Global?node-id=173-675&m=dev

Testing: 
1. Checkout the branch, yarn install, yarn start etc
2. Navigate to /en/social-innovation
3. Test the component for responsiveness, prettiness, accessibility etc etc

### CODE REVIEW

File: src/stylesheets/base/_buttons.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  margin-left: 8px;
+  margin-top: 20px;
+  padding: 12px 24px;
+  border-radius: 100px;
+  font-weight: 700;
+
+  &:hover {
+    background: #000;
+  }
+}
```
Comment (k-milbury) [inline src/stylesheets/base/_buttons.scss:17]: Ok sweet I hard coded that in so this is way better

File: src/stylesheets/slices/_featured_program.scss [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    @media screen and (min-width: $desktop){
+      max-width: 720px;
+		}
+	}
+	p {
+		margin-bottom: 30px;
+	}
+
+  .accordion {
+    background: rgba(#FFFFFF, 0.4);
```
Comment (k-milbury) [inline src/stylesheets/slices/_featured_program.scss:32]: should we add a variable for white?

**Review by k-milbury — APPROVED** [review]
Looking good to me and everything works as it should 😎, left one comment about a variable

File: src/stylesheets/slices/_featured_program.scss (hunk 2 above)
Comment (klancashire) [reply src/stylesheets/slices/_featured_program.scss:32]: We probably should, I actually just copied and pasted this from elsewhere in the project and didn't think about it haha.

File: src/stylesheets/base/_buttons.scss (hunk 1 above)
Comment (klancashire) [reply src/stylesheets/base/_buttons.scss:17]: Yeah feel free to steal this for your carousel! We're using this button everywhere.

**Pull request merged**